demo-shell/src/app/components/cloud/community/community-task-cloud.component.scss

```diff
@@ -0,0 +1,27 @@
+:host {
+    height: 100%;
+    min-height: 100%;
+    min-width: 100%;
+    width: 100%;
+}
+
+.adf-cloud-layout-tab-body {
+    .mat-tab-body-wrapper {
+        height: 100%;
+    }
+}
+
+.app-cloud-task-list-container {
+    place-content: stretch space-between;
+    align-items: stretch;
+    margin-top: 2px;
+}
+
+:host, .app-cloud-task-list-container {
```

Comment:
should better be scoped to the component root class, and have ViewEncapsulation.none

Reply:
done